shell/src/main/java/alluxio/cli/fs/command/StatCommand.java

```diff
@@ -94,7 +103,17 @@ protected void runPlainPath(AlluxioURI path, CommandLine cl)
   @Override
   public int run(CommandLine cl) throws AlluxioException, IOException {
     String[] args = cl.getArgs();
-    AlluxioURI path = new AlluxioURI(args[0]);
+    AlluxioURI path;
+    if (cl.hasOption("file-id")) {
```

Comment:
Make this string literal a static field `FILE_ID_OPTION` as in https://github.com/Alluxio/alluxio/blob/99e76a85141cdbdd7a96d13a6cc4a58c15728277/shell/src/main/java/alluxio/cli/fs/command/CpCommand.java#L90
and retrieve it by `FILE_ID_OPTION.getLongOpt()`.

Reply:
done